rbd/mirror.go

```diff
@@ -0,0 +1,167 @@
+// +build !mimic,!nautilus
```

Comment:
mimic has been dropped in the mean time, not need to mention it in the build-constraints anymore

Reply:
done